src/Sulu/Bundle/ContentBundle/Resources/public/js/components/content/main.js

```diff
@@ -943,6 +948,11 @@ define([
 
                     tabs: {
                         url: navigationUrl,
+                        options: {
+                            data: function() {
+                                return this.sandbox.util.extend(false, {}, this.content.toJSON());
```

Comment:
Why is the first parameter false? This way it is not a deep copy, is that what you want? And if no, why not use the `deepCopy` util function instead?